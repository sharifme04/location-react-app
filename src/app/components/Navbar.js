import React from "react";
import { AiOutlineMenu } from "react-icons/ai";
import logo from "../assets/icons/logo.png";

const Navbar = () => {
  return (
    <nav className="navbar navbar-dark fixed-top custom-style">
      <div className="container-fluid">
        <AiOutlineMenu className="menu-style" color={"#fff"} size={35} />
        <img
          src={logo}
          width="48"
          height="48"
          className="d-inline-block align-top"
          alt="logo"
        />
      </div>
    </nav>
  );
};

export default Navbar;
