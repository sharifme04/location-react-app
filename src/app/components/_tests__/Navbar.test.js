import React from "react";
import renderer from "react-test-renderer";
import Navbar from "../Navbar";

test("snapshot Form renders with empty props", () => {
  const component = renderer.create(<Navbar />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
