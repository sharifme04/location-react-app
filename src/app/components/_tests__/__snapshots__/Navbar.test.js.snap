// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot Form renders with empty props 1`] = `
<nav
  className="navbar navbar-dark fixed-top custom-style"
>
  <div
    className="container-fluid"
  >
    <svg
      className="menu-style"
      color="#fff"
      fill="currentColor"
      height={35}
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": "#fff",
        }
      }
      viewBox="0 0 1024 1024"
      width={35}
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M904 160H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0 624H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0-312H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8z"
      />
    </svg>
    <img
      alt="logo"
      className="d-inline-block align-top"
      height="48"
      src="logo.png"
      width="48"
    />
  </div>
</nav>
`;
